versions/2.0/vipps-login-migrate-api-1.0-to-2.0.md

```diff
@@ -0,0 +1,28 @@
+# Vipps Login API
+
+Document version 1.0.0.
+
+Reference:\
+[vipps-login-api-version-1.0](../../vipps-login-api)\
+[vipps-login-api-version-2.0](vipps-login-api.md)
+
+## Table of contents
+* [Overview](#overview)
+* [Convert to vipps login api version 2.0](#convert-to-vipps-login-api-version-20)
+
+## Overview:
+In vipps login api version 2.0 we are changing format of scope 'nnin' to 'nin'.
```

Comment:
Is this the right place to explain that we will minimise the information delivered as part of the ID-token, and that the user info endpoint will need to be used to get other information than the sub?

Some other suggested updates to the existing text:
In vipps login api version 2.0 we are changing format of scope 'nnin' to 'nin'.
We have also have a major change to the response from the '/userinfo' endpoint.
These are changes to the format of claim 'birthdate' from dd.mm.yyyy to yyyy-mm-dd to comply with the ISO 8601 format.
To comply with the OIDC-standard and to highlight the address that the user has set as default in Vipps the claim 'address' have been converted from list of all addresses to only return the users default address, 
and the none default addresses are served under 'other_address'.